db/migrate/20210323145018_add_email_tracking_to_school.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class AddEmailTrackingToSchool < ActiveRecord::Migration[6.1]
+  def change
+    change_table :schools, bulk: true do |t|
+      t.string   :nomination_email_token
+      t.datetime :nomination_email_sent_at
+      t.datetime :nomination_email_opened_at
+    end
```

Comment:
I think we might be better tracking this in a separate table. There is a mechanism to send this email several times, and I think we should keep all the data

Reply:
Completely agree